Mathlib/FieldTheory/IntermediateField/Algebraic.lean

```diff
@@ -97,6 +97,13 @@ theorem eq_of_le_of_finrank_eq' [FiniteDimensional F L] (h_le : F ≤ E)
     (h_finrank : finrank F L = finrank E L) : F = E :=
   eq_of_le_of_finrank_le' h_le h_finrank.le
 
+/-- Mapping a finite dimensional intermediate field along an algebra equivalence gives
+a finite-dimensional intermediate field. -/
+instance im_finiteDimensional {K L : Type*} [Field K] [Field L] [Algebra K L]
+    {E : IntermediateField K L} (σ : L ≃ₐ[K] L) [FiniteDimensional K E] :
```

Comment:
Why not make use of the existing variables line here? `variable {K L : Type*} [Field K] [Field L] [Algebra K L]`